doc/docs/.vuepress/public/specs/operateurs-api-v3.yaml

```
@@ -560,6 +560,12 @@ paths:
                      type: string
                      format: date-time
                      description: La date de création dans le registre
+                   terms_violation_errors:
+                     type: array
+                     description: | 
```

yamllint

[error] trailing spaces

(trailing-spaces)